slack-api-client/src/main/java/com/slack/api/methods/RequestFormBuilder.java

```diff
@@ -1134,7 +1135,7 @@ public static FormBody.Builder toForm(ChatScheduleMessageRequest req) {
         if (req.getBlocksAsString() != null) {
             form.add("blocks", req.getBlocksAsString());
         } else if (req.getBlocks() != null) {
-            String json = GSON.toJson(req.getBlocks());
+            String json = getJsonWithGsonAnonymInnerClassHandling(req.getBlocks());
```

Comment:
`req.getUsers(): List<String>` patterns also need the same workaround.

Reply:
Makes sense, I also noticed that the Map objects also suffered from this GSON issue, I added additional code to handle these use cases.